app/models/notification.rb

```diff
@@ -63,7 +63,8 @@ def self.types
                         watching_first_post: 17,
                         topic_reminder: 18,
                         liked_consolidated: 19,
-                        post_approved: 20
+                        post_approved: 20,
+                        commit_approved: 21
```

Comment:
To me this is probably the scope of the entire change. We just want the number reserved in core, the rest of the functionality / strings and so on can live in the plugin imo.

Reply:
Ok, I'll work out how to monkey-patch the other changes in.

Reply:
Prefer a less monkey patchy way, can you expose an API for it from the JS side?

Reply:
Ok